### ISSUE TITLE
Multer improperly imported

### ISSUE THREAD

**Issue by tambry** [thiagobustamante/typescript-rest#3]
Trying to compile the following:
```
import * as rest from "typescript-rest";
```

Will result in:
```
node_modules/typescript-rest/release/lib/server.d.ts(10,71): error TS2305: Module 'Express' has no exported member 'Multer'.
```

This looks like a bug in the typings not properly importing Multer. Any ideas?

**Comment by thiagobustamante** [comment]
Yes. You are right. There is a problem in the generated .d.ts file. I am fixing it now, but a fast work around is just import multer types in your project, like:

```
npm install --save-dev @types/multer
```

**Comment by thiagobustamante** [comment]
I fixed the dependencies and am declaring the @types/* dependencies now as main dependencies instead of dev dependencies on package.json file.

So, we don't need anymore to install these dependencies directly on our final project, as was previously described on the README file. I updated the README file and will publish these changes to npm repo.

Thank you!

**Comment by tambry** [comment]
Thank you for your quick response and fix! Unfortunately this breaks ES6 as a compilation target in TypeScript. See #4 and #5.

**Comment by thiagobustamante** [comment]
Oh....Sorry. I published 0.2.2 release to fix it. Thank you again

**Comment by tambry** [comment]
Thank you for the quick release!
